person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..1000)
```

Comment:
- [ OPTIONAL ] I think rand comes from a Class called Random so please add it before rand method to make random id works as expected ✨♻🔥💯